examples/labs/alex_controller/main.py

```diff
@@ -0,0 +1,14 @@
+from pitop import DriveController, PanTiltController, Camera
+from pitop.labs import AlexWebController
```

Comment:
Don't forget that this needs to change - we don't want classes with Alex, Bobby etc prefixes or suffixes